tools/runtime-stats-monitor.md

```diff
@@ -0,0 +1,65 @@
+The Runtime Net Stats Monitor can be configured from the inspector. You can add, remove, and customize its Display Elements in the inspector under Configuration/Display Elements. Each Display Element displays one or more stats, which can be added, removed, and modified under each Display Element’s “Stats” field.
+
+## Including/Excluding the Runtime Network Stats Monitor in Builds
+
+When using the tools package, the implementation of the Runtime Net Stats Monitor (RNSM) can be included/excluded from a build independently from its interface (the RuntimeNetStatsMonitor component), to enable developers to save space and other resources in cases where this is important, such as on mobile.
+
+The RNSM implementation is always included in the editor, is included by default in development builds, and is not included by default in release builds.
+
+### Excluding the RNSM from Development Builds
+
+The RNSM implementation is included in development builds by default. The RNSM can be excluded from development builds either by:
+
+Unchecking the “Include in Develop Builds” toggle under Project Settings > Multiplayer Tools > Net Stats Monitor.
+Manually defining the symbol UNITY_MP_TOOLS_NET_STATS_MONITOR_DISABLED_IN_DEVELOP under Project Settings > Player > Other Settings > Scripting Define Symbols.
+
+### Including the RNSM in Release Builds
+
+The RNSM implementation is not included in release builds by default. The RNSM can be included in release builds either by:
+
+Checking the “Include in Release Builds” toggle under Project Settings > Multiplayer Tools > Net Stats Monitor.
+Manually defining the symbol UNITY_MP_TOOLS_NET_STATS_MONITOR_ENABLED_IN_RELEASE under Project Settings > Player > Other Settings > Scripting Define Symbols.
+
+### Including the RNSM in All Builds
+
+In addition to the settings above that allow the RNSM to be included/excluded in development and release builds independently, the RNSM can be forcibly enabled in all builds by defining the symbol UNITY_MP_TOOLS_NET_STATS_MONITOR_IMPLEMENTATION_ENABLED under Project Settings > Player > Other Settings > Scripting Define Symbols. This option takes precedence over the options mentioned above to exclude the RNSM from development builds.
+
+## Generating Test Data
+
+To generate test data to display in the RNSM if your project does not already have a Netcode For Game Objects (NGO) Network Manager that is networking with clients, you may do the following:
+
+Define UNITY_MP_TOOLS_DEV under Project Settings > Player > Scripting Define Symbols.
+Create a new object.
+Attach either the “Test Data Generator Component” to this new object.
+Alternatively, to create test data that is more customizable, you can attach and configure the “Custom Test Data Generator” component.
+These two test data generation components can be safely used side-by-side, although this may be confusing as the data from both with be aggregated and displayed in the RNSM.
+
+## Styling the Runtime Network Stats Monitor
```

Comment:
@ian-m-unity @s-omeilia-unity we should add a section about using panel settings to address screen real estate issues